build_linux.sh

```diff
@@ -18,11 +20,11 @@ docker build --no-cache -t tor-brave -f Dockerfile-linux \
     --build-arg "openssl_hash=$OPENSSL_HASH" \
     --build-arg "tor_hash=$TOR_HASH" \
     ${1+"$@"} .
-docker run --rm --name tor-brave -d tor-brave
-docker cp "tor-brave:/tor-$TOR_VERSION/install/bin/tor" "tor-$TOR_VERSION-linux-brave-$BRAVE_TOR_VERSION"
+docker run --rm --name "$IMAGE_NAME" -d "$IMAGE_NAME"
```

Comment:
I think it's best to always use `--init` with `docker run` and `init: true` with `docker-compose`.

Not always necessary, but like with variable quoting - better to do it always and not leave room for mistakes.